### PR TITLE
DEX: Show price values converted to Money for last price and for trade estimate

### PR DESCRIPTION
**Pull request by rlucas6130**
## Ticket
* https://issues.aphelion-neo.com/tktview?name=f9e33a14d8

## Changes
* Updated right side of 'MarketSelection' component to match the mock.
* Update 'Estimated' label to be converted to USD 
* Added a 'Total (quoted asset)' Label and value above estimate label (to match mock).

Mock link: 
https://issues.aphelion-neo.com/raw/2fb80ac0a9575b6a2a5eb3c1b67d22336e25f6d12da648a08488b012386fff54?m=image/jpeg

## Screenshots
image

### CODE REVIEW

File: src/renderer/components/dex/MarketSelection.vue [hunk 1]
```diff
@@ -35,13 +35,16 @@
...
-            {{ $formatTokenAmount($store.state.tradeHistory.close24Hour) }}
-          </div>
-          <div :class="['change', {decrease: $store.state.tradeHistory.change24Hour < 0, increase: $store.state.tradeHistory.change24Hour > 0}]">{{ $formatNumber($store.state.tradeHistory.change24Hour) }}</div>
+        <div class="base-price">
+          {{ $formatTokenAmount($store.state.tradeHistory.close24Hour) }}
+        </div>
+        <div class="base-price-converted">
+          {{ $formatMoney($store.state.tradeHistory.close24Hour * baseCurrencyUnitPrice) }}
+        </div>
+        <span class="label">{{$t('change')}} ({{$store.state.currentMarket.quoteCurrency}})</span>
```
Comment (jsolman) [inline src/renderer/components/dex/MarketSelection.vue]: I know the mock just says ‘CHANGE (NEO)’, but we have feedback to switch to showing ‘24H CHANGE (NEO)’. You can switch the label to ‘change24H’ and the text to ‘24H CHANGE’

Comment (crishellco) [inline src/renderer/components/dex/MarketSelection.vue]: Please add spaces between curly braces and their expression.

```vue
<span class="label">{{ $t('change') }} ({{ $store.state.currentMarket.quoteCurrency }})</span>
```

File: src/renderer/components/dex/MarketSelection.vue [hunk 2]
```diff
@@ -60,6 +64,12 @@ export default {
     storeStateCurrentMarket() {
       return this.$store.state.currentMarket;
     },
+    baseCurrencyUnitPrice() {
+      return neo.getHolding(this.storeStateCurrentMarket.baseAssetId).unitValue;
```
Comment (crishellco) [inline src/renderer/components/dex/MarketSelection.vue]: All services are available via a mixin to each component.

```javascript
return this.$services.neo.getHolding(this.storeStateCurrentMarket.baseAssetId).unitValue;
```

File: src/renderer/components/dex/MarketSelection.vue (hunk 1 above)
Comment (rlucas6130) [reply src/renderer/components/dex/MarketSelection.vue]: I would recommend to insert all translated values in all lowercase for consistency. Then we can style it with text-transform: uppercase; (or whatever) to then apply transformations.

Comment (rlucas6130) [reply src/renderer/components/dex/MarketSelection.vue]: Nice catch. I wonder if there is linter rule we can add for this.

File: src/renderer/components/dex/MarketSelection.vue (hunk 2 above)
Comment (rlucas6130) [reply src/renderer/components/dex/MarketSelection.vue]: Fixed. Thanks for the help.

**Pull request merged**